source/index.html.md

```diff
@@ -1500,7 +1500,14 @@ It takes a single parameter, indicated as ”[location]” in the example, which
 | roundWithDateCount       | Number of rounds with a date available        |
 | reviewDate               | Date of latest company review by our analysts |
 
-
+## Deal search
+
+| Name               | Content                                                      |
+|--------------------|--------------------------------------------------------------|
+| results            | List of Section [Funding round](#deal)                       |
```

Comment:
Should it be (#deal)  or something else? 
Also the response of /getFundingRoundList includes List<FundingRoundES>, not the one we have already here. What to do with that?

Reply:
you should add the FundingRoundES model, just below the Startup one. Call it "Deal", and as such the "Content" of this should be "List of Section [Deal](#deal)"